types/index.ts

```diff
@@ -19,57 +19,119 @@ import { expectTypeOf } from 'expect-type'
+    [
+      [1, 2, 3],
+      [4, 5, 6],
+    ],
+    1,
+    'uncorrected'
+  )
   math.variance(1, 2, 3)
   math.variance([1, 2, 3])
-  math.variance([1, 2, 3], "biased")
-  math.variance([1,2, 3], 0, "biased")
-  math.variance([[1,2,3], [4,5,6]], 1, "unbiased")
-  math.variance([[1,2,3], [4,5,6]], 1, "uncorrected")
+  math.variance([1, 2, 3], 'biased')
+  math.variance([1, 2, 3], 0, 'biased')
+  math.variance(
+    [
+      [1, 2, 3],
+      [4, 5, 6],
+    ],
+    1,
+    'unbiased'
+  )
+  math.variance(
+    [
+      [1, 2, 3],
+      [4, 5, 6],
+    ],
+    1,
+    'uncorrected'
+  )
 
   // std and variance on chain
-  math.chain([1, 2, 3]).std("unbiased")
-  math.chain([[1, 2, 3], [4, 5, 6]]).std(0, "biased").std(0, "uncorrected")
-  math.chain([[1, 2, 3], [4, 5, 6]]).std(0, "biased").std(0, "uncorrected")
-  math.chain([1, 2, 3]).std("unbiased")
-  math.chain([[1, 2, 3], [4, 5, 6]]).variance(0, "biased")
-  math.chain([[1, 2, 3], [4, 5, 6]]).variance(1, "uncorrected").variance("unbiased")
-
+  math.chain([1, 2, 3]).std('unbiased')
+  math
+    .chain([
+      [1, 2, 3],
+      [4, 5, 6],
+    ])
+    .std(0, 'biased')
+    .std(0, 'uncorrected')
+  math
+    .chain([
+      [1, 2, 3],
+      [4, 5, 6],
+    ])
+    .std(0, 'biased')
+    .std(0, 'uncorrected')
+  math.chain([1, 2, 3]).std('unbiased')
+  math
+    .chain([
+      [1, 2, 3],
+      [4, 5, 6],
+    ])
+    .variance(0, 'biased')
+  math
+    .chain([
+      [1, 2, 3],
+      [4, 5, 6],
+    ])
+    .variance(1, 'uncorrected')
+    .variance('unbiased')
 
   // expressions
-  math.evaluate('1.2 * (2 + 4.5)');
+  math.evaluate('1.2 * (2 + 4.5)')
 
   // chained operations
-  const a = math.chain(3).add(4).multiply(2).done();
-  assert.strictEqual(a, 14);
+  const a = math.chain(3).add(4).multiply(2).done()
+  assert.strictEqual(a, 14)
 
   // mixed use of different data types in functions
-  assert.deepStrictEqual(math.add(4, [5, 6]), [9, 10]); // number + Array
-  assert.deepStrictEqual(math.multiply(math.unit('5 mm'), 3), math.unit('15 mm')); // Unit * number
-  assert.deepStrictEqual(math.subtract([2, 3, 4], 5), [-3, -2, -1]); // Array - number
-  assert.deepStrictEqual(math.add(math.matrix([2, 3]), [4, 5]), math.matrix([6, 8])); // Matrix + Array
+  assert.deepStrictEqual(math.add(4, [5, 6]), [9, 10]) // number + Array
+  assert.deepStrictEqual(
+    math.multiply(math.unit('5 mm'), 3),
+    math.unit('15 mm')
+  ) // Unit * number
+  assert.deepStrictEqual(math.subtract([2, 3, 4], 5), [-3, -2, -1]) // Array - number
+  assert.deepStrictEqual(
+    math.add(math.matrix([2, 3]), [4, 5]),
+    math.matrix([6, 8])
+  ) // Matrix + Array
 
   // narrowed type inference
-  const b: math.Matrix = math.add(math.matrix([2]), math.matrix([3]));
-  const c: math.Matrix = math.subtract(math.matrix([4]), math.matrix([5]));
+  const _b: math.Matrix = math.add(math.matrix([2]), math.matrix([3]))
+  const _c: math.Matrix = math.subtract(math.matrix([4]), math.matrix([5]))
 }
```

Comment:
Why did these underscores get added?

Reply:
Oh, I guess to flag that they are unused variables. I guess that's OK, but maybe comment the first instance of this? Just a thought.

Reply:
Yep, that's it.

I considered just removing the unused declarations, but decided that, without other nontrivial changes to this test file, removing the unused declarations would degrade readability (e.g., in scenarios where `a` through `e` are all declared, but `b` and `c` are unused, it would be weird to skip from `a` to `d`.)

IMO it's not worth adding a comment. `_` is a fairly common convention, if someone removes the underscore their IDE should give them get a linting hint, and if they add a new unused variable their IDE should give them a lintining hint about how to fix it.

Reply:
👍